src/Core/Domain/Order/QueryResult/OrderProductForViewing.php

```diff
@@ -43,6 +49,11 @@ class OrderProductForViewing
      */
     private $name;
 
+    /**
+     * @var array<OrderProductForViewing>
```

Comment:
Is that a new convention we started to use? I'm used to see `OrderProductForViewing[]` instead 🤔

Reply:
Fixed